src/pages/recipients.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import RecipientYear from '../components/recipientYear.js'
 import {graphql} from 'gatsby'
 import globalStyles from '../styles/global.module.css'
+import Footer from '../components/footer.js'
```

Comment:
I think you can start importing the Layout page to have both our header & footer. Unless you specifically wanted just the footer, I'm not sure what you're doing here.